lib/Migration/Version0017Date20200202112903.php

```diff
@@ -90,6 +89,7 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 			'instance', 'string', [
 						  'notnull' => false,
 						  'length'  => 255,
+						  'default' => '',
```

Comment:
Shouldn't this go into a new migration? Otherwise instances that have already created the table will stay not get a default set there.

Reply:
You're right, I'm not very familiar with Nextcloud migrations but I'll try to add a new migration instead of updating this one

Reply:
Done, but we can't use an empty string (`""`) as default value with the new migration, because it doesn't work with [`Doctrine\DBAL\Schema\Table::changeColumn()`](https://github.com/doctrine/dbal/blob/v2.10.1/lib/Doctrine/DBAL/Schema/Table.php#L320) method... You can check yourself if you want

Reply:
Wehn you put it into a new migration it would work, as it then drops and recreates it :-X

Reply:
@nickvergessen Sorry, I'm not sure to understand what you said...
But you can't drop the `instance` column, because it's part of the primary key.
I guess when the default value is `""`, `Doctrine\DBAL\Schema\Table::changeColumn()` think it's a null value, so the default value of the column is unchanged...
But with the `Doctrine\DBAL\Schema\Table::addColumn()` method it works, the value `""` isn't considered as a null value

Reply:
> But with the `Doctrine\DBAL\Schema\Table::addColumn()` method it works, the value `""` isn't considered as a null value

So I suppose, there is inconsistency between the `addColumn()` method and the `changeColumn()` method